python/cudf/cudf/_lib/cpp/nvtext/jaccard.pxd

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.column.column_view cimport column_view
+from cudf._lib.cpp.types cimport size_type
+
+
+cdef extern from "nvtext/jaccard.hpp" namespace "nvtext" nogil:
+
+    cdef unique_ptr[column] jaccard_index(
+        const column_view &input1,
```

Comment:
Does Cython support east const? :) (We don't have to change it in this PR, Cython const alignment isn't important to me, just wondering out loud whether it matters for Cython syntax...)